### DISCUSSION TITLE
Script-Mount the ISO from local storage

### DISCUSSION THREAD

**Discussion by ehsanijavad** [HewlettPackard/POSH-HPOneView#529] [Q&A] [answered]
Hi Chris
First of all, Thank you for your support.
Could you help me please?
I need a script:
I have created a server profile, now I want to open the console and mount the ISO (From my Laptop) and power it on.
again thank you for your answer and your support.

Regards
Jawad

**Comment by ChrisLynchHPE** [comment, marked as answer]
Check the scripts location [here](https://github.com/HewlettPackard/oneview-osdeployment/tree/master/PowerShell) on how to mount and unmount an ISO using iLO Virtual Media.  Keep in mind that this is a licensable feature, which means you need to have an iLO Advanced license, in order to script virtual media.

**Comment by ehsanijavad** [comment]
Thanks you so much Chris.
Please let me know, how can I mount ISO from my laptop (In script ISO resided in web server).
Thanks and Regards
Jawad

**Comment by ChrisLynchHPE** [comment]
You need to put the ISO into a directory where either an HTTP or FTP service can make it available.  This is all dependent upon what OS you are running.  Within Windows, you can install the HTTP Web Service role or feature.  Once it is installed and service is running, you'll need to add ISO as a MIME type supported by the web service.  There are plenty of blog articles on how to do this.

Once you have the URL of the ISO, use the script I referenced you to and replace the URL with your URL.

**Comment by ehsanijavad** [comment]
Thank you Chris.
